core/git_mixins/active_branch.py

```diff
@@ -34,22 +35,22 @@ def _get_branch_status_components(self):
         clean = len(addl_lines) == 0
 
         if first_line.startswith("## HEAD (no branch)"):
-            return True, False, None, None, clean, None, None
+            return True, False, None, None, clean, None, None, False
 
         if first_line.startswith("## Initial commit on "):
-            return False, True, first_line[21:], clean, None, None, None
+            return False, True, first_line[21:], clean, None, None, None, False
 
         valid_punctuation = "".join(c for c in string.punctuation if c not in "~^:?*[\\")
         branch_pattern = "[A-Za-z0-9" + re.escape(valid_punctuation) + "]+?"
-        short_status_pattern = "## (" + branch_pattern + ")(\.\.\.(" + branch_pattern + ")( \[((ahead (\d+))(, )?)?(behind (\d+))?\])?)?$"
+        short_status_pattern = "## (" + branch_pattern + ")(\.\.\.(" + branch_pattern + ")( \[((ahead (\d+))(, )?)?(behind (\d+))?(gone)?\])?)?$"
         status_match = re.match(short_status_pattern, first_line)
 
         if not status_match:
-            return False, False, addl_lines[0], None, clean, None, None
+            return False, False, addl_lines[0], None, clean, None, None, False
```

Comment:
Might be better to replace `addl_lines[0]` with `None if clean else addl_lines[0]` to prevent exceptions if there are additional cases where `short_status_pattern` doesn't match `first_line` but `addl_lines` is empty?